.github/workflows/main.yml

```diff
@@ -71,7 +71,11 @@ jobs:
         source $CONDA/etc/profile.d/conda.sh
         pkg=local::anaconda-anon-usage
         conda install anaconda-client constructor conda-index $pkg
-        conda create -p ./testenv $pkg conda==${{ matrix.cversion }} conda-libmamba-solver
+        if [[ ! "${{ matrix.cversion }}" =~ 4.* ]]; then

```

Comment:
Don't try to install libmamba for conda 4.*, and make sure to skip the libmamba tests below

Reply:
Good idea!